lib/watir-get-image-content/image.rb

```diff
@@ -7,13 +7,13 @@ class Image
     def to_x_base64(target)
       assert_exists
       js = %Q{var canvas = document.createElement("canvas");
-              canvas.width = arguments[0].width;
-              canvas.height = arguments[0].height;
+              canvas.width = arguments[1];
+              canvas.height = arguments[2];
               var ctx = canvas.getContext("2d");
               ctx.drawImage(arguments[0], 0, 0);
               var dataURL = canvas.toDataURL("image/#{target}");
               return dataURL.replace(/^data:image\\/(png|jpg|gif);base64,/, "");}
-      driver.execute_script js, @element
+      driver.execute_script js, @element, natural_width, natural_height
```

Comment:
@mmangino from where we got these variables?

Reply:
From watir: https://github.com/watir/watir/blob/master/lib/watir/elements/html_elements.rb#L681

I'm not sure if the naming has changed, but this code is working in production for us.